.travis.yml

```diff
@@ -84,15 +84,17 @@ script:
 - ./tools/lint.py
 - bash -c "sleep 2100; pkill ninja" &
 - ./tools/build.py -j2
-- ./tools/test.py $DENO_BUILD_PATH
+- TEMPFILE=$(mktemp)
+- /usr/bin/time -p ./tools/test.py $DENO_BUILD_PATH 2>$TEMPFILE
+- TEST_DURATION=$(cat $TEMPFILE | awk '/real/ {print $2}')
```

Comment:
This seems brittle - but I can't think of a better way to do this...

Reply:
`/usr/bin/time -p` should [guarantee](https://linux.die.net/man/1/time) following IEEE standard format, so not too bad I guess...

Reply:
tools/test.py could write its duration into the output directory - That would make it a bit more self contained.
My problem isn't so much parsing time output, but that the benchmark now depends on this little script in .travis.yml ... Like what happens if in the future we wanted to run benchmarks on windows? the awk dependency is going to mean we have to rewrite this...

Reply:
Idea:
1. Do the timing inside of ./tools/test.py. This seems like a good place: https://github.com/denoland/deno/blob/4c0517c33938ab56b3126b35330b56a91891940b/tools/test.py#L61
```python
import time
start = time.time()
print("hello")
end = time.time()
print(end - start)
```

2. Write the elapsed time to `out/debug/test_time.txt`. Define the filename in `tools/util.py` so it can be shared between `test.py` and `benchmark.py`.

3. Read that file in `tools/benchmark.py`. Ideally you'd have `readTestTime()` and `writeTestTime()` in `tools/util.py` and add a test for them in `tools/util_test.py`.